data/pigui/modules/new-game-window/start-variants.lua

```diff
@@ -0,0 +1,70 @@
+local Lang = require 'Lang'
+local lui = Lang.GetResource("ui-core")
+local Equipment = require 'Equipment'
+local SystemPath = require 'SystemPath'
+local misc = Equipment.misc
+local laser = Equipment.laser
+local Commodities = require 'Commodities'
+
+return
```

Comment:
For longer-term use, I'd recommend this module instead provide an API for other modules to register a starting preset, and a function to query the list of starting presets rather than returning a reference to this table.

Reply:
I did something, I don't know if that was meant.

Reply:
The new interface looks good, should let mods etc. register custom start presets!